styleguide/assets/icons/activity.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-activity"><polyline points="22 12 18 12 15 21 9 3 6 12 2 12"></polyline></svg>
```

Comment:
What is with all these icons? 😱

Are these from feather or are they custom?

Reply:
They were the original Feather SVGs from before we added it as a depency—I'll remove but we'll want to address how to add custom icons to the set in a separate issue.